packages/core/src/client.ts

```diff
@@ -588,6 +597,9 @@ export class MedplumClient extends EventTarget {
       this.autoBatchQueue = undefined;
     }
 
+    // if (this.authType === 'basic') {
```

Comment:
if CLI user switches to basic auth options, should we clear existing login so that there is no overlap between authentication?